app/_locales/en/messages.json

```diff
@@ -1673,6 +1676,17 @@
   "networkStatus": {
     "message": "Network status"
   },
+  "networkStatusBaseFeeTooltip": {
+    "message": "The base fee is set by the network and changes every 8-12 seconds. Our $1 and $2 options account for sudden increases.",
```

Comment:
I'm not sure this is accurate — what do we feel about saying "changes every 13-14 seconds on average"? (This number is derived from: https://etherscan.io/chart/blocktime)

Reply:
Hi @jakehaugen : can you plz suggest about above.

Reply:
Sounds good!

Reply:
I did the change 👍